blocks/test/fixtures/core-embed__vine.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:core-embed/vine {"url":"https://vine.com/"} -->
```

Comment:
I think these Vine files were inadvertently reintroduced after being removed in #4521.

Reply:
Merging the master branch into my PR branch should fix it, or should it not?

Reply:
No, I think it will re-add the files when merged.